### PR TITLE
[11.0][FIX] purchase_order_line_deep_sort: Fix compute order when order lines are created directly

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: purchase_order_line_deep_sort/models/purchase_order.py [hunk 1]
```diff
@@ -45,6 +46,8 @@ def resolve_subfields(obj, line_order):
         )
         for line in sorted_lines:
             sequence += 10
```
Comment (Tardo) [inline purchase_order_line_deep_sort/models/purchase_order.py:48]: Magic number...

**Review by Tardo — APPROVED** [review]

File: purchase_order_line_deep_sort/models/purchase_order.py (hunk 1 above)
Comment (pedrobaeza) [reply purchase_order_line_deep_sort/models/purchase_order.py:48]: Yes, as it's a suitable gap between lines. This is used in Odoo in other places. Any better suggestion?

Comment (Tardo) [reply purchase_order_line_deep_sort/models/purchase_order.py:48]: It's recommended to use a constant

Comment (pedrobaeza) [reply purchase_order_line_deep_sort/models/purchase_order.py:48]: Ah, well, I get you. OK, as this is out of this PR, let's merge it and we can improve it later with that constant for being in header.

**Pull request merged**